pulsar/datadog_checks/pulsar/check.py

```diff
@@ -0,0 +1,20 @@
+# (C) Datadog, Inc. 2022-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from datadog_checks.base import OpenMetricsBaseCheckV2
+
+from .config_models import ConfigMixin
+from .metrics import METRIC_MAP
+
+
+class PulsarCheck(OpenMetricsBaseCheckV2, ConfigMixin):
```

Comment:
Should OpenMetricsV1 still be supported since the original Pulsar supported that?

Reply:
No